HVCAENx527App/Db/HVCAENx527ch.substitutions

```diff
@@ -1,4 +1,4 @@
 file "$(TOP)/templates/HVCAENx527ch.tpl" {
 	pattern {PSNAME,CHANNUM,CHADDR,ALARM_WHEN_ON,ALARM_WHEN_RAMPING}
-	{"\$(P):\$(PSNAME):\$(SLOT)","\$(CHANNUM)","\$(CHADDR)","\$(ALARM_WHEN_ON)","\$(ALARM_WHEN_RAMPING)"}
+	{"\$(P):\$(PSNAME):\$(SLOT)","\$(CHANNUM)","\$(CHADDR)","\$(ALARM_WHEN_ON=NO_ALARM)","\$(ALARM_WHEN_RAMPING=NO_ALARM)"}
```

Comment:
The default is already specified in the `st.cmd`, I'm not sure it needs to be here as well?

Reply:
I saw and error or ",Undefined" appear somewhere which was why i added them - it might be to do with running in simulation mode, but i don't recall at the moment.